components/scream/src/control/atmosphere_driver.hpp

```diff
@@ -85,7 +85,8 @@ class AtmosphereDriver
   // which is handy for scream standalone tests.
   void initialize (const ekat::Comm& atm_comm,
                    const ekat::ParameterList& params,
-                   const util::TimeStamp& t0);
+                   const util::TimeStamp& t0,
+                   const bool restarted_run = false);
```

Comment:
Can we add some comments above this method that describe the parameters, since we're adding a new one? One can guess at what they mean, but we spend so much time guessing at things this way already. :-)

Reply:
Good point.